packages/app-mobile/components/screens/Notes/TextWrapCalculator.tsx

```
@@ -0,0 +1,52 @@
+import * as React from 'react';
+import { Platform, Text, ViewStyle } from 'react-native';
+
+interface Props {
+	textCompStyle: ViewStyle;
+	textCompContainerWidth: number; // Must be updated dynamically via onLayout of the container, in order for Text.onTextLayout to fire on every device rotation
+	showMultilineToggle: boolean | null;
+	multiline: boolean;
+	text: string;
+	updateState: (showMultilineToggle: boolean, multiline: boolean)=> void;
```

ESLint

[error] 'showMultilineToggle' is defined but never used. Allowed unused args must match /^_/u.

(no-unused-vars)